src/app/contact/contact.component.scss

```diff
@@ -1,18 +1,71 @@
-@import url('https://fonts.googleapis.com/css?family=Raleway');
-.contact__section {	
-    height: 225px;	
+@import "/home/srinivas/Desktop/AngularHomePage/angular-webpack2-starter/src/app/sassvariables/colorvariables.component.scss";
+@import "/home/srinivas/Desktop/AngularHomePage/angular-webpack2-starter/src/app/sassbreakpoint/sassbreakpoint.component.scss";
```

Comment:
use relative paths for import. search in google that difference between absolute path and relative path